### PR TITLE
New landing pages

### PR DESCRIPTION
**Pull request by lizonp**
screen shot 2018-04-19 at 11 35 52
screen shot 2018-04-19 at 11 36 12
screen shot 2018-04-19 at 11 36 28
screen shot 2018-04-19 at 11 36 39
screen shot 2018-04-19 at 11 36 48
screen shot 2018-04-19 at 11 36 54

### CODE REVIEW

File: app/components/shared/HorizontalLine.tsx [hunk 1]
```diff
@@ -3,10 +3,20 @@ import * as React from "react";
 
 import * as styles from "./HorizontalLine.module.scss";
 
+type TSize = "narrow" | "wide";
```
[the remark this answers is not in the document]
Comment (lizonp) [reply app/components/shared/HorizontalLine.tsx]: I have moved it to global types 👍

**Comment by lizonp** [comment]
screen shot 2018-04-23 at 10 31 57
screen shot 2018-04-23 at 10 07 42
@Moejoe90 page content is already placed in container.

File: app/components/Header.tsx [hunk 2]
```diff
@@ -9,6 +9,11 @@ import { appConnect } from "../store";
 import { appRoutes } from "./AppRouter";
 import * as styles from "./Header.module.scss";
 import { Button } from "./shared/Buttons";
+import { loginWalletRoutes } from "./walletSelector/walletRoutes";
+
+interface IProps {
+  loginLink?: string;
```
Comment (krzkaczor) [inline app/components/Header.tsx]: this is not ised currently

File: app/components/Header.tsx [hunk 3]
```diff
@@ -32,6 +37,16 @@ export const HeaderComponent: React.SFC<IStateProps & IDispatchProps> = props =>
       >
         LOGOUT
       </Button>
+    ) : (
+      <Link
+        to={
+          location.pathname === appRoutes.etoLanding ? appRoutes.loginEto : loginWalletRoutes.light
```
Comment (krzkaczor) [inline app/components/Header.tsx]: i would prefer if this is connected component, gets location from state and use selector to select login link

File: app/components/landing/shared/OffToOnChainCompany.tsx [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+      <div className={styles.steps}>
+        {steps.map(({ htmlTitle, htmlDescription }, index) => (
+          <div className={styles.step} key={index}>
+            <h4 className={styles.stepName} dangerouslySetInnerHTML={{ __html: htmlTitle }} />
+            <div className={styles.stepCount}>
+              <div className={styles.counter}>{index + 1}</div>
+            </div>
+            <p
+              className={styles.stepDescription}
+              dangerouslySetInnerHTML={{ __html: htmlDescription }}
```
Comment (krzkaczor) [inline app/components/landing/shared/OffToOnChainCompany.tsx]: please no `dangerouslySetInnerHTML` —  just use react elements to describe content.

File: app/components/landing/shared/OffToOnChainCompany.tsx [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+  return (
+    <div className={styles.offToOnChainCompany}>
+      <h3 className={styles.header}>
+        <strong>OFF-CHAIN</strong> company
+      </h3>
+      <HorizontalLine theme="yellow" size="narrow" />
+      <div className={styles.steps}>
+        {steps.map(({ htmlTitle, htmlDescription }, index) => (
+          <div className={styles.step} key={index}>
+            <h4 className={styles.stepName} dangerouslySetInnerHTML={{ __html: htmlTitle }} />
```
Comment (krzkaczor) [inline app/components/landing/shared/OffToOnChainCompany.tsx]: same here

**Pull request merged**